app/src/main/java/org/mozilla/fenix/components/FenixSnackbar.kt

```diff
@@ -86,8 +87,16 @@ class FenixSnackbar private constructor(
         /**
          * Display a snackbar in the given view with duration and proper normal/error styling.
          * Note: Duration is overriden for users with accessibility settings enabled
+         * displayedOnFragmentWithToolbar should be true for all snackbars that will end up
+         * being displayed on a BrowserFragment and must be true in cases where the fragment is
+         * going to pop TO BrowserFragment (e.g. EditBookmarkFragment, ShareFragment)
          */
-        fun make(view: View, duration: Int, isError: Boolean = false): FenixSnackbar {
+        fun make(
+            view: View,
+            duration: Int = LENGTH_LONG,
+            isError: Boolean = false,
+            isDisplayedOnBrowserFragment: Boolean
```

Comment:
should we give this a `false` default value so only the ones who need the param have to use it?